app/build.gradle

```diff
@@ -67,6 +67,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
     implementation 'com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava'
+    implementation 'com.google.firebase:firebase-database:19.3.0'
```

Comment:
We are not using firebase database, we are using firebase firestore and they are not compatible